src/Modules/Shared/Microsoft.PowerShell.Utility/Microsoft.PowerShell.Utility.psm1

```diff
@@ -737,4 +737,18 @@ function ConvertFrom-SddlString
             RawDescriptor = $rawSecurityDescriptor
         }
     }
-}
\ No newline at end of file
+}
+
+function Get-Uptime

```

Comment:
I'd prefer to see this as a cmdlet in C#.

For faster startup, we need to avoid PowerShell script in the core modules that are loaded in essentially every session.